src/pages/Upload/Upload.js

```diff
@@ -0,0 +1,210 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Upload = () => {
+  const imageLoad = FileName => {
+    return process.env.PUBLIC_URL + '/image/' + FileName + '.png';
```

Comment:
env파일에서 관리하지 않아도 될 것 같습니다.
```suggestion
    return '/image/' + FileName + '.png';
```
이렇게 하거나. 밑에서
`<ImgTag src='/image/seeMore.png alt="seeMore" />` 로 해주는 게 더 깔끔할 것 같긴 합니다!

그리고 env 파일에서는 image 관련 루트 뿐만아니라, key 같은 중요한 정보d의 환경 변수를 포함할 수 있기 때문에 보안상의 이유로 gitignore에 .env 파일을 등록해서 github상에 노출되지 않도록 해주세요